test/geometry_tests.cpp

```diff
@@ -61,105 +94,176 @@ bool checkFirstOrLastPoint(int firstOrLastPoint, int x_shouldbe, int y_shouldbe,
 
 }
 
+std::ostream& operator<<(std::ostream& os, const std::vector<Cell>& v){
+	os << "\n";
+	for (const auto& pixel:v) {
+		os << "(" << pixel.x << "," << pixel.y << ")\n";
+	}
+	os << "\n";
+	return os;
+}
+
+inline bool operator==(const Cell &lhs,const Cell &rhs) {
+	return (lhs.x == rhs.x) && (lhs.y == rhs.y);
+}
+
 TEST(Point2DPlanning_Griz_Test, bresenham_conversion_test) {
 	// Get the points for the line connecting (0,0) and (5,0)
-	auto v = bresenham_conversion(0, 0, 5, 0);
+	const auto v = bresenham_conversion(0, 0, 5, 0);
```

Comment:
Can we change this from `v` to `pixels` or something more descriptive?

Reply:
Done